coops-ui/package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "@babel/polyfill": "^7.0.0-rc.1",
     "axios": "^0.18.0",
+    "moment": "^2.24.0",
```

Comment:
Need to remove this and the file it's used in - ~ARFilingDates.vue

Reply:
OK.